### ISSUE TITLE
Provide a larger set of built-ins

### ISSUE THREAD

**Issue by Ramblurr** [metosin/malli#242]
As discussed, on slack, here are some primitive built-in schemas that would be great to have out of the box.

I didn't just make these up, rather, I sat down to try and use malli to model a domain in a new project and ran into these missing specs.

```
:int {:min 0 :max 10}
:double  {:min 0 :max 1}
:inst {:min #inst "2020-01-01" :max #inst "2020-02-01"}
```
min should be inclusive, and max exclusive (`min <= x < max`)

**Comment by lins05** [comment]
Also related: at allow easy registration of custom schemas.

e.g. `m/add-custom-schemas {:int int?, :double double, :whatever some-predicate}`

Currently it requires a new custom registry and use it in every api call.

**Comment by rschmukler** [comment]
@lins05 there's actually support for both mutable registries and dynamically bound registries - making it so you don't need to continuously pass the arguments around. Take a look at the docs for the [mutable registries](https://github.com/metosin/malli#mutable-registry) and [dynamic registries](https://github.com/metosin/malli#dynamic-registry)

**Comment by lins05** [comment]
That's good information, thx @rschmukler .

**Comment by ikitommi** [comment]
Merged in #249. There is now `m/-simple-schema` which allows easy way to create custom schemas that use properties in validation and two helpers to do range-validation using `:min` and `:max` properties. The new builtins: `:double`,  `:boolean`, `:keyword`,  `:symbol`,  `:qualified-keyword`, `:qualified-symbol` and `:uuid`

not sure if having **both** predicate schemas (`int?`, `keyword?`) and type-schemas (`:int`, `:keyword`) in the default registry is a good idea. It's easy to pick either when using a custom registry, but what should be the default?

related issue: #25

**Comment by ikitommi** [comment]
as the dates already have an issue (#49), will close this.
